src/Integration.Vsix/SonarLintDaemon/SonarLintDaemon.cs

```diff
@@ -28,19 +28,21 @@
 using System.Linq;
 using System.Net;
 using Grpc.Core;
-using Microsoft.VisualStudio.Shell;
 using Sonarlint;
 
 namespace SonarLint.VisualStudio.Integration.Vsix
 {
     [Export(typeof(ISonarLintDaemon))]
-    class SonarLintDaemon : ISonarLintDaemon
+    internal class SonarLintDaemon : ISonarLintDaemon
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Fix this implementation of 'IDisposable' to conform to the dispose pattern. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS3881)